Bio/SeqIO/SnapGeneIO.py

```diff
@@ -51,7 +54,7 @@ def _iterate(handle):
         yield (packet_type, length, data)
 
 
-def _parse_dna_packet(length, data, record):
+def _parse_dna_packet(length, data, record, _):
```

Comment:
Yes.

To be frank I’m not entirely happy with this solution. A cleaner design IMHO would have been for all those functions to be methods of the `SnapGeneIterator` class, so that all the state could be held within the class without having to pass it around between functions. But I felt this was too much of a refactoring for a simple bugfix.